modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -12,58 +12,63 @@
 
     $_env_configs                        = {
       'dep'  => {
-        worker_group             => 'dep-pushapk',
-        worker_type              => 'dep-pushapk',
-        verbose_logging          => true,
-        taskcluster_client_id    => secret('pushapk_scriptworker_taskcluster_client_id_dep'),
-        taskcluster_access_token => secret('pushapk_scriptworker_taskcluster_access_token_dep'),
-        scope_prefix             => 'project:releng:googleplay:',
-        cot_product              => 'firefox',
-
-        sign_chain_of_trust      => false,
-        verify_chain_of_trust    => true,
-        verify_cot_signature     => false,
+        worker_group               => 'dep-pushapk',
```

Comment:
Sorry for the indentation change, `do_not_contact_google_play` is just a super fat string :annoyed: